test/integration.test.js

```diff
@@ -22,11 +23,20 @@ describe('css-vendor', () => {
       expect(supportedProperty('display')).to.be('display')
     })
 
-    it('should prefix if needed', () => {
-      const prop = supportedProperty(`${prefix.css}animation`)
-      if (prop !== 'animation') {
-        expect(prop).to.be(`${prefix.css}animation`)
-      }
+    browser("requires prefix for transform property", {
```

Comment:
You take specific browsers, where you know the outcome and you do an assertion with expected output.  You selectively run an assertion. Not really a hack, but could be better: we run always the assertion and compare the results with the result expected in a certain browser. Without maintaining the versions etc. Just get the result for the current browser from caniuse (not http api, there is a static version) and then compare it with what we got from  the browser.